pod-rotation-controller/values.yaml

```diff
@@ -0,0 +1,128 @@
+## Mina Foundation image version
+##
+## @param image.repository [default:  673156464838.dkr.ecr.us-west-2.amazonaws.com/github-actions-runner] pod-rotation-controller image name
+## @param image.pullPolicy pod-rotation-controller image pull policy
+## @param image.tag [default: default-2023.09.26] tag name
+image:
+  repository: 673156464838.dkr.ecr.us-west-2.amazonaws.com/github-actions-runner
```

Comment:
I am using this image as it has both kubectl and jq
Because both packages will need to be upgraded, I found it easier to use this image instead of creating a new one